traffic_portal/app/src/common/modules/form/deliveryService/form.deliveryService.anyMap.tpl.html

```diff
@@ -51,7 +51,7 @@
                     <hr class="divider"/>
                     <li role="menuitem"><a ng-click="viewCharts()">View Charts</a></li>
                     <hr class="divider"/>
-                    <li role="menuitem"><a ng-click="viewServers()">Manage Servers</a></li>
+                    <li ng-if="::(!deliveryService.topology)" role="menuitem"><a ng-click="viewServers()">Manage Servers</a></li>
```

Comment:
IMO these should be !(deliveryservice.Topology instanceof Object)

Reply:
Nah
```javascript
const a = {topology: "test"};
console.log(a.topology instanceof Object); // false
```

Reply:
Oh, that's just the name. `typeof deliveryService/Topology !== 'string'` in that case

Reply:
But then you could have
```javascript
const a = {topology: ""};
console.log(typeof a.topology === "string"); //true
```

Reply:
True, the loose comparison is useful in this case.